drivers/net/ethernet/renesas/rswitch_tc_flower.c

```diff
@@ -391,6 +415,11 @@ static int rswitch_tc_flower_setup_action(struct rswitch_tc_filter *f,
 			f->action |= ACTION_CHANGE_DMAC;
 			rswitch_parse_pedit(f, act);
 			break;
+		case FLOW_ACTION_VLAN_MANGLE:
+			f->action = ACTION_VLAN_CHANGE;
```

Comment:
It's better to write
f->action |= ACTION_VLAN_CHANGE;
Because, for example, we can write in tc command changing DMAC then VLAN mangle. In this case DMAC will not be changed.

Reply:
Done